.ci_support/win_64_cuda_compilernvcccuda_compiler_version11.2.yaml

```diff
@@ -1,17 +0,0 @@
-c_compiler:
```

Comment:
Appears the changes in `skip` resulted in the Windows builds being dropped. Is that intended?

Reply:
Yes, see PR description.

> Support for Windows CUDA 11.x has been dropped in nvcomp 3. This PR also drops that package. CUDA 12 for Windows is not yet available, but this recipe can be updated so that build is not skipped and it should work fine once support is added.